main.py

```diff
@@ -91,7 +91,7 @@ def main():
 
         # Puts a small delay here to address a problem with Mapillary not registering the access_token immediately after
         # distributing it
-        time.sleep(5)
+        time.sleep(2)
```

Comment:
Reduction here since 5s was too much and it was a bit of a pain while developing.